src/schemas/entry.yaml

```
@@ -15,7 +15,10 @@ properties:
   category_id:
     type: integer
     example: 5
+  Authors: 
```

```suggestion
  authors: 
```
Please stick to _lower_snake_case_.